hw/ip/otbn/doc/_index.md

```diff
@@ -490,6 +490,35 @@ The error that caused the alert can be determined by reading the {{< regref "FAT
 If OTBN was running, this value will also be reflected in the {{< regref "ERR_BITS" >}} register.
 A fatal alert can only be cleared by resetting OTBN through the `rst_ni` line.
 
+
+### Error Prioritization
+
+OTBN can signal multiple errors at once, by setting multiple bits of {{< regref "ERR_BITS" >}}.
+There are some scenarios where one error takes priority over another.
+In these scenarios the higher priority error will suppress some other errors.
+
+* Any error that produces a fatal alert will suppress all non fatal errors.
```

Comment:
```suggestion
* Any fatal error supresses all recoverable errors.
```

I think the terminology we have been using so far is that an "error" is the condition that is detected, an "alert" is an action resulting from an error. (There might be more actions, e.g. OTBN-internal error handling.)